ee/dtr/admin/manage-jobs/audit-jobs-via-ui.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: Audit Jobs via the Web Interface
+title: Audit Jobs with the web interface
```

Comment:
Would it be better to say "web UI" here?  The "old" (and still useful) way of doing it is through the REST API, which is still a web interface.